### PR TITLE
ENT-3273 | Don't show order history in dropdown if user is associated…

### PR DESCRIPTION
**Pull request by iloveagent57**
… with a subscription.

https://openedx.atlassian.net/browse/ENT-3273

User that has an active enterprise learner portal:
Screen Shot 2020-08-18 at 11 37 10 AM

User that does _not_ have an active enterprise learner portal:
Screen Shot 2020-08-18 at 11 37 44 AM

### CODE REVIEW

**Review by bbaker6225 — APPROVED** [review]
LGTM once tests pass

**Review by adamstankiewicz — APPROVED** [review]

**Comment by iloveagent57** [comment]
Tagging @edx/teaching-and-learning for review, please.

**Review by davidjoy — DISMISSED** [review]
I'm wondering if we can package `useEnterpriseConfig` up and expose it from `@edx/frontend-enterprise` as a hook.

I see that we have the same code in https://github.com/edx/frontend-component-header-edx/blob/master/src/Header.jsx#L22, and I think it may be common for us to start having app-specific headers and this is likely to come up again.  

In addition to letting you all more easily update the code in that hook, it will also create a lighter coupling between the MFEs and the `@edx/frontend-enterprise` code.  Ideally I think we'd only add `@edx/frontend-enterprise` to MFEs through a plugin for edx.org, but we just don't have the framework to do that yet. In the meantime, I think it'd behoove us to keep them as loosely coupled as we can.

**Comment by iloveagent57** [comment]
@davidjoy makes sense, I'll give it a shot.

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,4 @@ temp/babel-plugin-react-intl
 /.vscode
 
 # Local package dependencies
-frontend-enterprise/
+module.config.js
```
Comment (adamstankiewicz) [inline .gitignore:21]: Should `module.config.js` be .gitignored?

File: webpack.dev.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { getBaseConfig } = require('@edx/frontend-build');
+let config = getBaseConfig('webpack-dev');
+// const path = require('path');
+
+// config.resolve.alias = {
+//   react: path.resolve('./node_modules/react'),
+// };
+
+module.exports = config;
```
Comment (adamstankiewicz) [inline webpack.dev.config.js]: I think this file can be deleted now.

Comment (iloveagent57) [reply webpack.dev.config.js]: yes

File: README.rst [hunk 3]
```diff
@@ -43,3 +43,28 @@ In this project, install requirements and start the development server by runnin
    npm start # The server will run on port 1995
 
 Once the dev server is up, visit http://localhost:2000/course/course-v1:edX+DemoX+Demo_Course to view the demo course.  You can replace ``course-v1:edX+DemoX+Demo_Course`` with a different course key.
+
+Local module development
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To develop locally on modules that are installed into this app, you'll need to create a ``module.config.js``
+file (which is git-ignored) that defines where to find your local modules, for instance::
```
Comment (adamstankiewicz) [inline README.rst:51]: nit: extra colon after "instance"

Comment (adamstankiewicz) [reply README.rst:51]: Ignore me :) RST syntax.

File: README.rst [hunk 4]
```diff
@@ -43,3 +43,28 @@ In this project, install requirements and start the development server by runnin
...
+
+     moduleName: the name you use to import code from the module.
+     dir: The relative path to the module's source code.
+     dist: The sub-directory of the source code where it puts its build artifact.  Often "dist".
+     */
+     localModules: [
+        { moduleName: '@edx/paragon/scss', dir: '../paragon', dist: 'scss' },
+        { moduleName: '@edx/paragon', dir: '../paragon', dist: 'dist' },
+        { moduleName: '@edx/frontend-enterprise', dir: '../frontend-enterprise', dist: 'src' },
+        { moduleName: '@edx/frontend-platform', dir: '../frontend-enterprise', dist: 'dist' },
```
Comment (adamstankiewicz) [inline README.rst]: nit: `../frontend-platform`?

File: .env.development [hunk 5]
```diff
@@ -4,6 +4,7 @@ BASE_URL='localhost:2000'
 CREDENTIALS_BASE_URL='http://localhost:18150'
 CSRF_TOKEN_API_PATH='/csrf/api/v1/token'
 ECOMMERCE_BASE_URL='http://localhost:18130'
+ENTERPRISE_LEARNER_PORTAL_HOSTNAME='localhost:8734'
```
Comment (adamstankiewicz) [inline .env.development:7]: should this include `http://`?

**Review by adamstankiewicz — APPROVED** [review]
LGTM. Just one small nit.

**Pull request merged**